src/host/outputStream.cpp

```diff
@@ -24,39 +24,6 @@ ConhostInternalGetSet::ConhostInternalGetSet(_In_ IIoProvider& io) :
 {
 }
 
-// Routine Description:
-// - Handles the print action from the state machine
-// Arguments:
-// - string - The string to be printed.
-// Return Value:
-// - <none>
-void ConhostInternalGetSet::PrintString(const std::wstring_view string)
-{
-    auto dwNumBytes = string.size() * sizeof(wchar_t);
-
-    auto& cursor = _io.GetActiveOutputBuffer().GetTextBuffer().GetCursor();
-    if (!cursor.IsOn())
-    {
-        cursor.SetIsOn(true);
-    }
-
-    // Defer the cursor drawing while we are iterating the string, for a better performance.
-    // We can not waste time displaying a cursor event when we know more text is coming right behind it.
-    cursor.StartDeferDrawing();

```

Comment:
/note: where'd this go

Reply:
oh duh 

> The root cause was the fact that we called cursor.StartDeferDrawing()
> before outputting the text, and this was something I had adopted in the
> new implementation as well. But I've now removed that, and instead just
> call cursor.SetIsOn(false). This seems to avoid the cursor droppings,
> and hopefully still has similar performance benefits.